roles/create_pr/README.md

```diff
@@ -13,6 +13,7 @@ partner_email                       | undefined | true/false                  |
 target_repository                   | undefined | true                        | GitHub repository where the pull request will be created.
 product_type                        | undefined | true                        | Product type. Either "helmchart" or "operator".
 publish_chart                       | true      | false                       | For helm charts, disable adding the Chart as part of the pull request.
+git_protocol                        | ssh       | false                       | Protocol for GitHub communication (ssh/https)
```

Comment:
maybe a nice to have is to include a validation in the main.yml to make sure `git_protocol` contains either `ssh` or `https`, wdyt?